Sources/Fuzzilli/Core/JavaScriptEnvironment.swift

```diff
@@ -766,11 +791,81 @@ public extension ObjectGroup {
             "prototype" : .object()
         ],
         methods: [
-            "resolve"    : [.anything] => .jsPromise,
-            "reject"     : [.anything] => .jsPromise,
-            "all"        : [.object()] => .jsPromise,
-            "race"       : [.object()] => .jsPromise,
-            "allSettled" : [.object()] => .jsPromise,
+            "resolve"    : [.anything...] => .jsPromise,
```

Comment:
Are these parameter types correct? From what I can tell, `resolve` and `reject` take one (arbitrary) argument (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/resolve) and `all` and similar methods expect an array of Promises as arguments: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

Reply:
yes, this is incorrect. I've fixed them in the latest commit.